account-keys/src/domain_separators.rs

```diff
@@ -1,4 +1,7 @@
 // Copyright (c) 2018-2020 MobileCoin Inc.
 
+/// The tag for the signature context
+pub const FOG_AUTHORITY_SIGNATURE_TAG: &[u8; 23] = b"Fog authority signature";
```

Comment:
Just curious: Is there a particular reason to use `&[u8; 23]` instead of `&str`?

Reply:
Not really, this is just what we merged before